resources/views/account/requestable-assets.blade.php

```diff
@@ -29,97 +29,40 @@
-                                                        <td>{{ trans('admin/hardware/general.requestable') }}</td>
-                                                    @endif
-
-                                                    <td>{{ $asset->expected_checkin }}</td>
-                                                    <td>
-                                                        <form action="{{route('account/request-item', ['itemType' => 'asset', 'itemId' => $asset->id])}}" method="POST" accept-charset="utf-8">
-                                                        {{ csrf_field() }}
-                                                        @if ($asset->isRequestedBy(Auth::user()))
-                                                            {{Form::submit(trans('button.cancel'), ['class' => 'btn btn-danger btn-sm'])}}
-                                                        @else
-                                                            {{Form::submit(trans('button.request'), ['class' => 'btn btn-primary btn-sm'])}}
-                                                        @endif
-                                                        </form>
-                                                    </td>
-
+                                                <th class="col-md-1" data-field="image" data-formatter="imageFormatter" data-sortable="true">{{ trans('general.image') }}</th>
+                                                <th class="col-md-2" data-field="model" data-sortable="true">{{ trans('admin/hardware/table.asset_model') }}</th>
+                                                <th class="col-md-2" data-field="model_number" data-sortable="true">{{ trans('admin/models/table.modelnumber') }}</th>
+                                                <th class="col-md-2" data-field="name" data-sortable="true">{{ trans('admin/hardware/form.name') }}</th>
+                                                <th class="col-md-3" data-field="serial" data-sortable="true">{{ trans('admin/hardware/table.serial') }}</th>
+                                                <th class="col-md-2" data-field="location" data-sortable="true">{{ trans('admin/hardware/table.location') }}</th>
+                                                <th class="col-md-2" data-field="status" data-sortable="true">{{ trans('admin/hardware/table.status') }}</th>
+                                                <th class="col-md-2" data-field="expected_checkin" data-formatter="dateDisplayFormatter" data-sortable="true">{{ trans('admin/hardware/form.expected_checkin') }}</th>
+                                                <th class="col-md-1" data-formatter="assetRequestActionsFormatter" data-field="actions" data-sortable="false">{{ trans('table.actions') }}</th>
                                             </tr>
-                                            @endforeach
-                                        </tbody>
+                                        </thead>
```

Comment:
Bootstrap tables! So much nicer.

Reply:
I still have to do it for requestable asset models, but that means more Bootstrap Tables formatters and ugh.